modules/models.py

```diff
@@ -35,14 +34,15 @@
     ds_config = generate_ds_config(shared.args.bf16, 1 * world_size, shared.args.nvme_offload_dir)
     dschf = HfDeepSpeedConfig(ds_config) # Keep this object alive for the Transformers integration
 
+
 def load_model(model_name):
     print(f"Loading {model_name}...")
     t0 = time.time()
 
     shared.is_RWKV = model_name.lower().startswith('rwkv-')
 
     # Default settings
-    if not any([shared.args.cpu, shared.args.load_in_8bit, shared.args.load_in_4bit, shared.args.gptq_bits > 0, shared.args.auto_devices, shared.args.disk, shared.args.gpu_memory is not None, shared.args.cpu_memory is not None, shared.args.deepspeed, shared.args.flexgen, shared.is_RWKV]):
```

Comment:
I'm reluctant to remove `--load-in-4bit` because that will certainly cause confusion, but I guess we can do it and move on.

Reply:
It's up to you but I think that it's a bad practice to keep multiple arguments which do the same thing

Reply:
You have convinced me, let's ditch `--load-in-4bit`.